apps/system/utils/utils_cpuload.c

```diff
@@ -169,9 +182,15 @@ static void cpuload_print_pid_value(char *buf, void *arg)
 #ifdef CONFIG_SMP
 			char * avgload[CONFIG_SMP_NCPUS + 1];
 			avgload[0] = stat_info[i];
-			for (int j = 0; j < CONFIG_SMP_NCPUS + 1; j++) {
+			for (int j = 0; j < CONFIG_SMP_NCPUS; j++) {
 				avgload[j] = strtok_r(avgload[j], "-", &avgload[j + 1]);
-				printf(" %5s |", avgload[j]);
+				if (cpu < CONFIG_SMP_NCPUS) {
+					if (cpu == j) {
+						printf(" %5s |", avgload[j]);
+					}
+				} else {
+					printf(" %5s |", avgload[j]);
+				}
```

Comment:
Here and line205~211 have the same flow. You can ignore my suggestion but I think we should do the same way for the same.

Reply:
Fixed. Thanks.